packages/react-next/src/utilities/ButtonGrid/ButtonGrid.test.tsx

```diff
@@ -43,7 +44,7 @@ describe('ButtonGrid', () => {
   });
   it('Can render a ButtonGrid with width of 2', () => {
     const wrapper = shallow(
-      <ButtonGrid
+      <ButtonGridBase
```

Comment:
Just curious, why are these being run with ButtonGridBase? That's not what we do for most of the component tests.

Reply:
I investigated this and it is a bit of a strange scenario. It seems after switching `ButtonGridBase` to `ButtonGrid` and shallow to safeMount, the tests fail because the gridcell role on the `DefaultButton` is called 32 times on both the v7 and v8 versions of the test. What is strange is if you console.log(wrapper.html) the gridcell role only exists eight times. After switching `DefaultButton` to button, all of the tests suddenly pass. I switched both versions to use `ButtonGrid` but I’m not entirely sure of what caused the error with `DefaultButton`.